roles/osp/admin-keystone-domain/tasks/main.yml

```diff
@@ -1,5 +1,30 @@
 ---
 
+- name: "Add EPEL temporary repository (but leave it disabled)"
+  yum_repository:
+    name: tmp-epel
+    description: EPEL YUM repo
+    baseurl: http://download.fedoraproject.org/pub/epel/7/$basearch
+    metalink: https://mirrors.fedoraproject.org/metalink?repo=epel-7&arch=$basearch
+    gpgcheck: no
+    enabled: no
+
+- name: "Install python-pip from EPEL"
+  yum:
+    name: python-pip
+    enablerepo: tmp-epel
+    state: present
+
+- name: "Install Python Shade using pip"
+  pip:
+    name: shade
+
```

Comment:
Would it be good to uninstall `python-pip` here?

Reply:
no - prefer to keep it